cybersixgill_actionable_alerts/README.md

```diff
@@ -0,0 +1,57 @@
+# Agent Check: cybersixgill_actionable_alerts
+
+## Overview
+
+Cybersixgill actionable alerts provides an automated threat intelligence solution based on comprehensive data sources 
+from the deep, dark and surface web. It is customizable, enabling you to define key assets relevant to your brand, 
+industry, and geolocation. This integration provides an out-of-the-box dashboard to monitor, prioritize, and respond to 
+critical assets such as IP addresses, domains, vulnerabilities, and VIPs.
+
+This check monitors [cybersixgill_actionable_alerts][1].
+
+## Setup
+
+### Installation
+
+To install the cybersixgill_actionable_alerts check on your host:
+1. Install the [developer toolkit][11] on any machine.
+2. To build the package, run the command: `ddev release build cybersixgill_actionable_alerts`
+3. [Install the Datadog Agent][10] on your host.
+4. Once the Agent is installed, upload the build artifact by running the command: `datadog-agent integration install -w
+ path/to/cybersixgill_actionable_alerts/dist/datadog_cybersixgill_actionable_alerts-0.0.1-py3-none-any.whl`.
+
+### Configuration
+1. Provide Client Id and Client Secret in Configuration.yaml file which you will get it once you contact info@cybersixgill.com.
```

Comment:
Can you provide more insight on this process on acquiring the client id and secret? Does the customer need to provide you information to retrieve it?

Reply:
Yes, Cybersixgill has many products so depending on the Integration user will get the proper client id and client secret.